<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-19T22:57:58 IST" name="Smoke" finished-at="2023-01-19T22:58:06 IST" duration-ms="8618">
    <groups>
      <group name="Smoke">
        <method signature="MSG91CheckBalance.msg91CheckBalance(java.lang.String)[pri:0, instance:com.nitara.MSG91.MSG91CheckBalance@7133da86]" name="msg91CheckBalance" class="com.nitara.MSG91.MSG91CheckBalance"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2023-01-19T22:57:58 IST" name="Smoke" finished-at="2023-01-19T22:58:06 IST" duration-ms="8618">
      <class name="com.nitara.MSG91.MSG91CheckBalance">
        <test-method signature="msg91CheckBalance(java.lang.String)[pri:0, instance:com.nitara.MSG91.MSG91CheckBalance@7133da86]" started-at="2023-01-19T22:57:58 IST" name="msg91CheckBalance" finished-at="2023-01-19T22:58:06 IST" duration-ms="8587" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[https://api.msg91.com]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[For input string: "{"msg":"418","msgType":"error"}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: For input string: "{"msg":"418","msgType":"error"}"
at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
at java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
at java.base/java.lang.Double.parseDouble(Double.java:651)
at com.nitara.MSG91.MSG91CheckBalance.msg91CheckBalance(MSG91CheckBalance.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- msg91CheckBalance -->
      </class> <!-- com.nitara.MSG91.MSG91CheckBalance -->
    </test> <!-- Smoke -->
  </suite> <!-- Smoke -->
</testng-results>
